tsinfer/inference.py

```diff
@@ -490,6 +607,111 @@ def match_samples(
     return ts
 
 
+def map_additional_mutations(
```

Comment:
What about ``insert_missing_sites`` as a more descriptive name?

Reply:
Could do. But I see this as useful to a general user, not just for our internal purposes. For instance, Wilder has clipped out the triallelic and low-quality sites from his inferences, but we could use this function to put them back in. So the sites are not "missing" as such, it's just that we suddenly find that we might want to append some new ones to the original set.

Reply:
They're missing from the tree sequence, right?

Reply:
Yes, that's a fair point.